### PR TITLE
Fix for #247

### PR DESCRIPTION
**Pull request by vkatsuba**
Related to https://github.com/AdRoll/rebar3_format/issues/247

### CODE REVIEW

File: src/formatters/default_formatter.erl [hunk 1]
```diff
@@ -471,7 +471,10 @@ lay_no_comments(Node, Ctxt) ->
                     {_, none} ->
                         lay(N, Ctxt1);
                     {_, [Arg]} ->
-                        case erl_syntax:type(Arg) == atom andalso erl_syntax:concrete(Arg) == ignore
+                        {attr, ModAttr, _, _} = erl_syntax:get_attrs(Arg),
+                        case erl_syntax:type(Arg) == atom
+                             andalso erl_syntax:concrete(Arg) == ignore
+                             andalso lists:member({text, "module"}, ModAttr) =/= true
                         of
```
Comment (elbrujohalcon) [inline src/formatters/default_formatter.erl:478]: You can use Tag directly, like…
```erlang
{Tag, [Arg]} ->
    case Tag /= module
         andalso erl_syntax:type(Arg) == atom
         andalso erl_syntax:concrete(Arg)
    of
        true -> …
```

**Review by elbrujohalcon — CHANGES_REQUESTED** [review]
A tiny improvement.

File: src/formatters/default_formatter.erl (hunk 1 above)
Comment (vkatsuba) [reply src/formatters/default_formatter.erl]: Yep, better. Done. Thanks.

**Comment by vkatsuba** [comment]
> A tiny improvement.

Done.

**Review by elbrujohalcon — APPROVED** [review]
Beautiful!

**Pull request merged**